Formula/php.rb

```diff
@@ -37,11 +39,6 @@ class Php < Formula
   needs :cxx11
 
   def install
-    # Ensure that libxml2 will be detected correctly in older MacOS
-    if MacOS.version == :el_capitan || MacOS.version == :sierra
```

Comment:
Looks like this may still be needed for Sierra:
```
ld: file not found: /usr/lib/system/libsystem_darwin.dylib for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
make: *** [ext/opcache/opcache.la] Error 1
make: *** Waiting for unfinished jobs....
```
🙃